framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/constants/images/Image.java

```diff
@@ -32,6 +32,7 @@
     CONTROLLER(OpenShiftConstants.KIE_IMAGE_TAG_CONTROLLER),
     KIE_SERVER(OpenShiftConstants.KIE_IMAGE_TAG_KIE_SERVER),
     MYSQL(OpenShiftConstants.KIE_IMAGE_TAG_MYSQL),
+    OPERATOR(OpenShiftConstants.KIE_IMAGE_TAG_OPERATOR),
```

Comment:
Where is this enum used?

Reply:
In ImageStreamProvider class. E.g. [here](https://github.com/kiegroup/kie-cloud-tests/pull/579/files#diff-d14038884f3545bd9904c7decf0b83bbR49-R51)

Reply:
In ImageStreamProvider class. E.g. [here](https://github.com/kiegroup/kie-cloud-tests/pull/579/files#diff-d14038884f3545bd9904c7decf0b83bbR49-R51)

Reply:
ok, thanks